projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
I wrote the changes which were the reason of the last PR (so I am the author of the changes) but I didn't create the file. What are we gonna do now? I don't care if my name is in the file or not. I would put the old authors in combined with OpenElec substituted with LibreELEC substitutions. What do you think guys?

@CvH I think Codesnake meant that he didn't do the changes in the last PR.